mirror.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: OpenHarmony Mirror
+parent: Eclipse Oniro Project
+---
+
+# OpenHarmony Mirror
+
+The Eclipse Oniro project offers a full OpenHarmony mirror of all its Gitee
+repositories on [GitHub](https://github.com/eclipse-oniro-mirrors/).
+All of these mirrored repositories are one-way, and only sync changes from Gitee
+to GitHub once a day.
+
+The reason for this mirror is the increased speed and reliability it offers
+to the Eclipse Oniro project and its developers. We encourage to use it to avoid
+disconnects during sync and slow download speeds for developers working outside of China.
+In addition to speed and reliability it also allows to streamline the process
```

Comment:
In addition to speed and reliability it also allows for streamlining the process